docs/libcurl/curl_ws_send.md

```diff
@@ -99,18 +80,29 @@ calls.
 ~~~c
 #include <string.h> /* for strlen */
 
-const char *send_payload = "magic";
+const char *buffer = "magic";
 
 int main(void)
 {
   size_t sent;
+  size_t sent_total = 0;
   CURLcode res;
   CURL *curl = curl_easy_init();
+
   curl_easy_setopt(curl, CURLOPT_URL, "wss://example.com/");
   curl_easy_setopt(curl, CURLOPT_CONNECT_ONLY, 2L);
   curl_easy_perform(curl);
-  res = curl_ws_send(curl, send_payload, strlen(send_payload), &sent, 0,
-                     CURLWS_PING);
+
+  while (res && buflen > 0) {
```

Comment:
~~~
docs/libcurl/curl_ws_send.3:70:17: error: ‘buflen’ undeclared (first use in this function); did you mean ‘buffer’?
   70 |   while (res && buflen > 0) {
      |                 ^~~~~~
      |                 buffer
docs/libcurl/curl_ws_send.3:70:17: note: each undeclared identifier is reported only once for each function it appears in
docs/libcurl/curl_ws_send.3:70:8: warning: while with space (SPACEBEFOREPAREN)
   while (res && buflen > 0) {
        ^